modules/nf-core/happy/ftxpy/tests/main.nf.test

```diff
@@ -72,8 +72,7 @@ nextflow_process {
             assertAll(
                 { assert process.success },
                 { assert snapshot(
-                    process.out.features,
-                    process.out.versions
+                    process.out
```

Comment:
Can you just put the process.out on one line (for consistency with other modules)